src/main/resources/org/jenkinsci/plugins/saml/SamlSecurityRealm/config.jelly

```diff
@@ -16,5 +16,36 @@
     <f:entry title="Maximum Authentication Lifetime" field="maximumAuthenticationLifetime" help="/plugin/saml/help/maximumAuthenticationLifetime.html">
       <f:textbox default="86400" />
     </f:entry>
+    <f:entry title="Username Attribute"  field="usernameAttributeName" 
+             help="/plugin/saml/help/usernameAttributeName.html">
+      <f:textbox />
+    </f:entry>
+    
+    <f:optionalBlock title="Use Encryption ?" 
+                     field="encryptionData"
+    checked="${instance.encryptionData != null}"
+                     help="/plugin/saml/help/encryption.html">
+      <f:entry title="Keystore path"  field="keystorePath" help="/plugin/saml/help/keystorePath.html">
+        <f:textbox />
+      </f:entry>
+      <f:entry title="Keystore password"  field="keystorePassword" help="/plugin/saml/help/keystorePassword.html">
+        <f:textbox />
+      </f:entry>
+      <f:entry title="Private Key password"  field="privateKeyPassword" help="/plugin/saml/help/keyPassword.html">
+        <f:textbox />
+      </f:entry>
+    </f:optionalBlock>
+    
+    <!-- 
```

Comment:
Looks like temporary commented out block that needs to be removed before merging to main branch.